src/main/java/com/google/maps/OkHttpRequestHandler.java

```diff
@@ -57,32 +60,36 @@ public OkHttpRequestHandler() {
 
     return new OkHttpPendingResult<T, R>(req, client, clazz, fieldNamingPolicy, errorTimeout);
   }
+  @Override
```

Comment:
Seems that you have dropped a bunch of blank lines here. Can you put them back please?